Feature 1

ShiroYaksha90 opened the pull request:
Add linters
Add HTML file
Add CSS file

ShiroYaksha90 committed:
Added linters file

ShiroYaksha90 committed:
Added linter-config files

ShiroYaksha90 committed:
Edited README.md

ShiroYaksha90 committed:
Added html file

ShiroYaksha90 committed:
Added Css file

ShiroYaksha90 committed:
Added test.md and Modified .gitignore

ShiroYaksha90 committed:
Update README.md

ShiroYaksha90 committed:
Update README.md file

ShiroYaksha90 committed:
Fix stylelint bugs

ShiroYaksha90 committed:
Merge branches 'Feature-1' and 'Feature-1' of https://github.com/ShiroYaksha90/Hello-Microverse into Feature-1

Abenezer-Tilahun commented on styles.css:
```diff
@@ -0,0 +1,10 @@
+  background-color: rgb(164, 255, 225);
+}
+
+h1 {
+  color: blue;
+  text-align: center;
+  margin: inherit;
+}
```
- [x] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
![Capture](https://user-images.githubusercontent.com/93598189/197765215-b499f339-633e-4a34-a65f-8a5b9bbd5558.PNG)

ShiroYaksha90 replied:
Thank you for your generous feedback I made the changes you suggested.

Abenezer-Tilahun reviewed (changes requested):
Hi @ShiroYaksha90   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No Linter Error
✔️ You used correct GitHub Flow.
✔️ Good README 
✔️ Good commit messages

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️

- [x] Your  README file should be descriptive all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo
- [x] Your Pull Request should have a descriptive title telling you what changes are inside and have a short summary describing introduced changes in more detail. To update the Pull request kindly click on 3 dot.

![Capture](https://user-images.githubusercontent.com/93598189/197839066-ff2adf74-3b7f-477f-8449-a95c54aabc66.PNG)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ShiroYaksha90 committed:
Updated READ.md

Add all the main headings from 
[readme-template] (https://github.com/microverseinc/readme-template/blob/master/README.md)
Add more contact info

ShiroYaksha90 committed:
Updated README.md

Remove unnecessary headings

ShiroYaksha90 committed:
Add class head to HTML and CSS file

ShiroYaksha90 committed:
Merge branch 'Feature-1' of https://github.com/ShiroYaksha90/Hello-Microverse into Feature-1

Abenezer-Tilahun reviewed (approved):
Hi @ShiroYaksha90 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
